src/components/Tooltip/index.js

```diff
@@ -0,0 +1,56 @@
+// @noSnapshot
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import { positionType } from './types'
+import { Bubble, ChildrenContainer, Container } from './styles'
+import Message from './exposedStyles/Message'
+
+class Tooltip extends PureComponent {
+  state = {
+    showTooltip: false,
+  }
+
+  onShowTooltip = () => this.setState({ showTooltip: true })
+
+  onHideTooltip = () => this.setState({ showTooltip: false })
+
+  render() {
+    const { children, message, position } = this.props
+    const { showTooltip } = this.state
+
+    return (
+      <Container onMouseLeave={this.onHideTooltip}>
+        <Bubble position={position} showTooltip={showTooltip}>
+          <Message>{message}</Message>
+        </Bubble>
+        <ChildrenContainer onMouseOver={this.onShowTooltip} onFocus={this.onShowTooltip}>
+          {children}
+        </ChildrenContainer>
+      </Container>
+    )
+  }
+}
+
+Tooltip.defaultProps = {
+  position: 'top',
+}
+
+Tooltip.propTypes = {
+  /* The children on which the tooltip will be displayed over. */
+  children: PropTypes.oneOfType([PropTypes.element, PropTypes.node]).isRequired,
+  /* The message to display in the tooltip. */
+  message: PropTypes.string.isRequired,
```

Comment:
Are you sure we should restrict it to string only ? What about having some DOM in it for example ?

Reply:
Good point. I added the option to pass a dom element